warp/examples/interop/example_stf_cg.py

```
@@ -0,0 +1,330 @@
+
+@wp.kernel
+def update_cond(
+    cond: wp.array[int],
+    iter_count: wp.array[int],
+    rs_new: wp.array[float],
+    tol2: float,
+    max_iters: int,
+):
+    """Increment iter_count; set cond=1 iff we should keep iterating."""
+    iter_count[0] = iter_count[0] + 1
+    keep = int(0)
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)